doc/dev/developer_guide/debugging-gdb.rst

```diff
@@ -0,0 +1,43 @@
+GDB - The GNU Project Debugger
+==============================
+
+`The GNU Project Debugger (GDB) <https://www.sourceware.org/gdb>`_ is
+a powerful tool that allows you to analyze the execution flow
+of a process.
+GDB can help to find bugs, uncover crash errors or track the
+source code during execution of a development cluster.
+It can also be used to debug Teuthology tests runs.
+
+GET STARTED WITH GDB
+--------------------
+
+Basic commands usage with examples can be found `here. <https://geeksforgeeks.org/gdb-command-in-linux-with-examples>`_
+GDB can be attached to a running process. For instance, after deploying a
+development cluster, the pid of ceph-osd deamon can be found in::
+
```

Comment:
s/commands//
s/pid/process number (PID)/
s/ceph-osd deamon/a \`\`ceph-osd\`\` daemon

Reply:
This is nitpicky, but `ceph-osd` should be delimited by two backticks \`\` not a double-quote char.

Reply:
Updated, Thank you!